packages/cashier/src/stores/account-transfer-store.ts

```diff
@@ -719,8 +719,6 @@ export default class AccountTransferStore {
             const { is_ok, message } = validNumber(converter_to_amount, {
                 type: 'float',
                 decimals: getDecimalPlaces(currency || ''),
-                min: 0,
-                max: 0,
```

Comment:
Shouldn't this be

min: this.transfer_limit.min,
max: this.transfer_limit.max

Reply:
@nijil-deriv no, `this.transfer_limit.min`  and `this.transfer_limit.max` we use to show min and max amount for `from_amount` field, for `to_amount` in crypto transfer we don't show a hint with min and max amount. I can add this hint if needed. But current production behaviour is that we don't show hint for `to_amount` field.
<img width="401" alt="Screenshot 2023-02-22 at 16 05 52" src="https://user-images.githubusercontent.com/103181646/220629173-756e9461-3323-4117-bea7-00651d61330f.png">

Reply:
Ah yes, thank you for clarifying.